apps/client/components/ui/BackToTopButton.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
+
+import { useEffect, useState } from 'react'
+import { BiArrowBack } from 'react-icons/bi'
+
+export const BackToTopButton = () => {
+  const [isVisible, setIsVisible] = useState(false)
+
```

Comment:
To wszystko powinno być w custom hooku, np. useScrollToTop, możesz go zdefiniować nawet nad tym komponentem w tym samym pliku (jest o tym mowa w filmie, który ci wysłałem).